allennlp/modules/augmented_lstm.py

```diff
@@ -67,6 +70,19 @@ def __init__(self,
         else:
             self.input_linearity = torch.nn.Linear(input_size, 4 * hidden_size, bias=True)
             self.state_linearity = torch.nn.Linear(hidden_size, 4 * hidden_size, bias=True)
+        self.reset_parameters()
+
+    def reset_parameters(self):
+        # Use sensible default initializations for parameters.
+        block_orthogonal(self.input_linearity.weight.data, [self.hidden_size, self.input_size])
+        block_orthogonal(self.state_linearity.weight.data, [self.hidden_size, self.hidden_size])
+
+        self.input_linearity.bias.data.fill_(0.0)
+        self.state_linearity.bias.data.fill_(0.0)
+        # Initialize forget gate biases to 1.0 as per An Empirical
+        # Exploration of Recurrent Network Architectures, (Jozefowicz, 2015).
+        self.input_linearity.bias.data[self.hidden_size:2 * self.hidden_size].fill_(1.0)
```

Comment:
I saw some formatting in @matt-peters's code for this that I liked: `data[self.hidden_size:(2 * self.hidden_size)]`.  It's a lot easier to read when you have slice indices with spaces.